uxarray/grid/utils.py

```diff
@@ -229,3 +229,47 @@ def _newton_raphson_solver_for_gca_constLat(init_cart,
         _iter += 1
 
     return np.append(y_new, constZ)
+
+
+def _get_face_edge_connectivity_cartesian(face_nodes_ind, face_edges_ind,
```

Comment:
@aaronzedwick 

I'd like to add that the name "Face Edge Connectivity Cartesian" is confusing, since Connectivity variables do not care about what coordinate system they are in. 

We can think of `face_edge_connectivity`, which has shape `(n_face, n_max_face_nodes)` contains the indices into each edge that composes each face. We store the nodes that make up each edge in `edge_node_connectivity`, which itself has a shape `(n_node, 2)`

One way to think about it is through this "execution/evaluation" order

`face_edge_connectivity` (index into) $\implies$ `edge_node_connectivity` (index into) $\implies$ `cartesiain_xyz_coords`


We start with the edges of each face, we then break down each edge into it's node indicies, lastly we index the desired coordinates to obtain the values of the coordinates at each edge node. 

Does this make sense?

Reply:
A better name would be, since we are utilizing multiple connectivity variables. 

`_get_cartesiain_face_edge_nodes`

Reply:
That does make sense, and I agree that name is better.